src/components/PersonRow/PersonRow.tsx

```diff
@@ -0,0 +1,18 @@
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+import { Link } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonRow: React.FC<Props> = ({ person }) => {
```

Comment:
i think the name PersonLink would be more clear

Reply:
Good point, fix it